hashicorp/terraform-k8s
# Decouple the code from the terraform version.

Instead of parsing the state in order to extract the outputs we call the
TFE API to get the same information.  That way we don't need to pull
terraform as a dependency.



#### How to test
Create a workspace as usual and verify in TFC that the workspace is using the latest version of TF available.

Additionally set the key `terraformVersion` in  your workspace definition to set an older terraform version.
